src/examples/js/CartClientExample.js

```diff
@@ -54,4 +59,14 @@ console.log(
 
 console.log('\n');
 
+console.log(
+  JSON.stringify(
+    await cartClientExample.completeOrderById('YOUR_CART_ID', {
+      paymentMethod: 'YOUR_PAYMENT_METHOD',
```

Comment:
Just add a comment here that `YOUR_PAYMENT_METHOD` can be one of `COD` or `PREPAID`